dev/traffic_router/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -o errexit
+trap '[ $? -eq 0 ] && exit 0 || echo "Error on line ${LINENO} of ${0}"; exit 1' EXIT
+
+cd "$TC/traffic_router"
+
+mvn -Dmaven.test.skip=true compile -P \!rpm-build
+mvn -Dmaven.test.skip=true package -P \!rpm-build
+
+/opt/tomcat/bin/catalina.sh jpda run
```

Comment:
One would think so. I'm still trying to get that to work.

Reply:
It does work if you upgrade to OpenJDK 11, remove the demo webapps, and put `ROOT.war` in the webapps directory

Reply:
I had done that, and it would start but it was not servicing any HTTP or DNS requests I threw at it. At the time we weren't using Java 11 - originally I had it trying to do that but when it wasn't working I downgraded to our at-the-time-supported Java version. It can be upgraded to 11 safely now, but I remain skeptical that it will have any significant impact on TR running.

Reply:
Responded at <https://github.com/apache/trafficcontrol/pull/6532#issuecomment-1027131794>. The TR changes are sufficient for TR itself to work, the missing pieces are outside of TR.